apps/a_dndtoggle/a_dndtoggle.app.js

```diff
@@ -29,15 +34,15 @@ switch (current) {
 	
 console.log("new: " + bSettings.quiet);
 
-E.showMessage(modeNames[current] + " -> " + modeNames[bSettings.quiet]);
+E.showMessage(modeNames[current] + " -> \n" + modeNames[bSettings.quiet]);
 setTimeout('exitApp();', 2000);
 
 
 function exitApp(){
 
-require("Storage").writeJSON("setting.json", bSettings);
-// reload clocks with new theme, otherwise just wait for user to switch apps
-
+  require("Storage").writeJSON("setting.json", bSettings);
+	// reload clocks with new theme, otherwise just wait for user to switch apps
+	
```

Comment:
Looks like some whitespace change has accidentally made it it

```suggestion
require("Storage").writeJSON("setting.json", bSettings);
// reload clocks with new theme, otherwise just wait for user to switch apps

```